rq/connections.py

```diff
@@ -11,6 +11,19 @@ class NoRedisConnectionException(Exception):
 
 @contextmanager
 def Connection(connection: t.Optional['Redis'] = None):  # noqa
+    """The context manager for handling connections in a clean way.
+    It will push the connection to the LocalStack, and pop the connection
+    when leaving the context
+    Example:
+        ..codeblock:python::
+
+            with Connection():
+                w = Worker()
+                w.work()
+
+    Args:
+        connection (Optional[Redis], optional): A Redis Connection instance. Defaults to None.
+    """    
     if connection is None:
```

Comment:
Can we add a deprecation warning here?

Reply:
Sure. I'm just leaving this PR as is, and continue the docstrings on a new one. Don't want to fall behind master too much. Already added that on the new one, so I'll resolve this for now.